### PR TITLE
Fix scaffold

### PR DESCRIPTION
**Pull request by pepibumur**
### Short description 📝
I tried scaffold in a project generated with `tuist init`, and I noticed that the template that we place under `Templates` lacks one of the attributes.
I fixed that and added an acceptance test that ensures that we can scaffold a project with that template, and compile it.

Moreover, I took the opportunity to set up the workflow to run acceptance tests in parallel.

### CODE REVIEW

File: Templates/default/ExampleTemplate.stencil [hunk 1]
```diff
@@ -1,12 +1,15 @@
 import ProjectDescription
 let nameAttribute: Template.Attribute = .required("name")
+let platformAttribute: Template.Attribute = .optional("platform", default: "iOS")
```
Comment (fortmarek) [inline Templates/default/ExampleTemplate.stencil]: There is no need to declare an extra variable, I'd move `.optional(...)` to `attributes` to make it a little bit simpler.

File: features/scaffold.feature [hunk 2]
```diff
@@ -6,4 +6,12 @@ Feature: Scaffold a project using Tuist
...
-    Then content of a file named TemplateProject/generated.swift in a directory TemplateProject should be equal to // Generated file with platform: ios and name: TemplateProject
\ No newline at end of file
+    Then content of a file named TemplateProject/generated.swift in a directory TemplateProject should be equal to // Generated file with platform: ios and name: TemplateProject
+  
+  Scenario: The project is a just initialized project
+    Given that tuist is available
+    And I have a working directory
+    And I initialize a ios application named MyApp
+    And tuist scaffolds a framework template to Projects/ named MyFeature
+    When tuist generates the project at Projects/MyFeature
```
Comment (fortmarek) [inline features/scaffold.feature]: Well done, thanks for doing this 👏

File: features/scaffold.feature [hunk 3]
```diff
@@ -6,4 +6,12 @@ Feature: Scaffold a project using Tuist
     Then I copy the fixture ios_app_with_templates into the working directory
     Then tuist scaffolds a custom template to TemplateProject named TemplateProject
     Then content of a file named TemplateProject/custom.swift in a directory TemplateProject should be equal to // this is test TemplateProject content
-    Then content of a file named TemplateProject/generated.swift in a directory TemplateProject should be equal to // Generated file with platform: ios and name: TemplateProject
\ No newline at end of file
+    Then content of a file named TemplateProject/generated.swift in a directory TemplateProject should be equal to // Generated file with platform: ios and name: TemplateProject
+  
+  Scenario: The project is a just initialized project
```
Comment (pepibumur) [inline features/scaffold.feature]: @fortmarek this is the test that I added

File: Templates/default/ExampleProject [hunk 4]
```diff
@@ -1,12 +1,4 @@
...
-                                 product: .framework,
-                                 productName: "{{ name }}",
-                                 bundleId: "io.tuist.{{ name }}",
-                                 infoPlist: .default,
-                                 sources: "Sources/**",
-                                 dependencies: []),
-])
+import ProjectDescriptionHelpers
+
+let project = Project.framework(name: "{{ name }}", platform: .{{ platform }}, dependencies: [])
```
Comment (fortmarek) [inline Templates/default/ExampleProject]: I love this ❤️  Good idea leveraging `ProjectDescriptionHelpers`

**Pull request merged**